logging.txt

```diff
@@ -0,0 +1,35 @@
+Request method:	POST
+Request URI:	https://reqres.in/api/register
+Proxy:			<none>
+Request params:	<none>
+Query params:	<none>
+Form params:	<none>
+Path params:	<none>
+Headers:		Accept=*/*
+				Content-Type=application/json
+Cookies:		<none>
+Multiparts:		<none>
+Body:
+{
+    "email": "eve.holt@reqres.in",
```

Comment:
la ce ai nevoie de. acest fisier?

Reply:
Aici doar am vrut sa vad cum functioneaza logarea requestului si a responsului in fisier txt. cu filters in RestAssured.